index.js

```diff
@@ -3700,6 +3700,26 @@
   S.pairs =
   def('pairs', {}, [$.StrMap(a), $.Array($.Pair($.String, a))], pairs);
 
+  //# insert :: String -> a -> StrMap a -> StrMap a
+  //.
+  //. Inserts a given property key and value into a shallow clone of a given
+  //. string map. Overrides the property if it already exists.
```

Comment:
Let's avoid using the term _property_ in relation to string maps. The fact that string maps are unadorned objects is an implementation detail in most contexts.

Adding the following would help those coming from Ramda who are likely to expect the function to be named `assoc`:

_Equivalent to Haskell's `insert` function. Similar to Clojure's `assoc` function._